.github/workflows/ci-tests.yml

```diff
@@ -54,12 +54,15 @@ jobs:
       - name: NPM version
         run: npm -v
 
-      - name: Install js-controller
-        run: npm ci
+      - name: Prepare build
+        run: npm i --ignore-scripts # install typescript and @types do not `setup first`
 
       - name: Build TS files
         run: npm run build
 
+      - name: Install js-controller
+        run: npm ci
+
```

Comment:
I would have done it the other way round... `npm ci --ignore-scripts` and later `npm install`, which should just run the install scripts.